rudder-core/src/main/scala/com/normation/rudder/domain/reports/ReportType.scala

```diff
@@ -104,13 +104,17 @@ object ReportType {
         case (_, BadPolicyMode)       | (BadPolicyMode, _)        => BadPolicyMode
```

Comment:
ha, i'm not sure I want to have by default the auditnotapplicable
Can i simply switch the last two lines ?